src/Symfony/Component/Validator/Constraints/NullValidator.php

```diff
@@ -18,6 +18,8 @@
  * @author Bernhard Schussek <bschussek@gmail.com>
  *
  * @api
+ *
+ * @deprecated
```

Comment:
Is it really required to deprecate this one also? Deprecations always have a cost for end users, the less the better to me.